release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -91,6 +91,24 @@ For more information about enabling a feature set during installation, see xref:
 [id="ocp-4-12-post-installation"]
 === Post-installation configuration
 
+[id="ocp-4-12-csi-driver-install-vsphere"]
+==== CSI driver installation on vSphere clusters
+To install a CSI driver on a cluster running on vSphere, the following requirements must be met:
+
+* VMware vSphere version 7.0.2 or later
+
+* vCenter 7.0.2 or later
+
+* Virtual machines of hardware version 15 or later
+
+* No third-party CSI driver already installed in the cluster
++
+If a third-party CSI driver is present in the cluster, {product-title} does not overwrite it.
+
+Components with versions earlier than those above are still supported, but are deprecated. These versions are still fully supported, but version 4.12 of {product-title} requires vSphere virtual hardware version 15 or later.
+
+Failing to meet the above requirements prevents {product-title} from upgrading to {product-title} 4.13 or later.
```

Comment:
@lisa I see https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-installation-vsphere-csi mentioned the same as below, could we use the same to keep the consistency. (And add the upgrade reminder you wrote last sentence.)


```
To install a CSI driver on a cluster running on vSphere, you must have the following components installed:

Virtual hardware version 15 or later

vSphere version 7.0 Update 2 or later

VMware ESXi version 7.0 Update 2 or later

Components with versions earlier than those above are deprecated or removed. Deprecated versions are still fully supported, but Red Hat recommends that you use vSphere 7.0 Update 2 or later and ESXi 7.0 Update 2 or later.

For more information, see [Deprecated and removed features](https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-deprecated-removed-features).
```

Reply:
@duanwei33 I've changed the order for the 4.12 RN to match 4.11 RN. As to the upgrade reminder, are you asking me to update the 4.11 RN to have that reminder? Because the 4.12 RN does have the upgrade statement.

Reply:
@lpettyjo I think we don't need to update 4.11 if there is no complaint.

Besides the order, I wonder if we could add the following statement like 4.11:
```
For more information, see [Deprecated and removed features](https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-deprecated-removed-features).
```
After
```
Components with versions earlier than those above are still supported, but are deprecated. These versions are still fully supported, but version 4.12 of {product-title} requires vSphere virtual hardware version 15 or later.
```
It will be more clear to check the deprecated and removed version.
(Sorry I did not make it clear in my initial comment)

Reply:
Done.

Reply:
Thanks for the update.
/lgtm